config/routes.rb

```diff
@@ -111,6 +112,11 @@
       end
     end
 
+    resources :sar_internal_review, only: only, controller: 'sar_internal_review', as: :case_sar_internal_review do
+      get '/(:step)', on: :collection, to: 'sar_internal_review#new', as: 'step'
+      post '/sar_internal_review/(:number)', on: :collection, to: 'sar_internal_review#start_internal_review', as: 'start_internal_review'
```

Comment:
are you going to add button to allow user to add SAR IR directly from standard SAR? if not, you don't need to the second path

Reply:
Good spot I added this in when I was working out the code and then realised that the `#start_internal_review` method wasn't needed for the MVP so I will remove 👍